tests/integration/test_logcollector/test_localfile/test_invalid_agent_localfile_config.py

```diff
@@ -0,0 +1,192 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: The 'wazuh-logcollector' daemon monitors configured files and commands for new log messages. When Wazuh is
+       configured incorrectly then a configuration error is displayed in the Wazuh's log, and Wazuh does not start (if
+       it has been restarted previously).
+
+tier: 0
+
+modules:
+    - logcollector
+
+components:
+    - agent
+
+daemons:
+    - wazuh-logcollector
+
+os_platform:
+    - linux
+    - windows
+
+os_version:
+    - Amazon Linux 1
+    - Amazon Linux 2
+    - Arch Linux
+    - Debian Buster
+    - Debian Stretch
+    - Debian Jessie
+    - Debian Wheezy
+    - CentOS 6
+    - CentOS 7
+    - CentOS 8
+    - Fedora 31
+    - Fedora 32
+    - Fedora 33
+    - Fedora 34
+    - openSUSE 42
+    - Oracle 6
+    - Oracle 7
+    - Oracle 8
+    - Red Hat 6
+    - Red Hat 7
+    - Red Hat 8
+    - Solaris 10
+    - Solaris 11
+    - SUSE 12
+    - SUSE 13
+    - SUSE 14
+    - SUSE 15
+    - Ubuntu Bionic
+    - Ubuntu Trusty
+    - Ubuntu Xenial
+    - Ubuntu Focal
+    - macOS Server
+    - macOS Sierra
+    - macOS Catalina
+    - Windows XP
+    - Windows 7
+    - Windows 8
+    - Windows 10
+    - Windows Server 2003
+    - Windows Server 2012
+    - Windows Server 2016
+    - Windows Server 2019
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/log-data-collection/index.html
+    - https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/localfile.html#location
+    - https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/localfile.html#log-format
+
+tags:
+    - logcollector
+'''
+import os
+import tempfile
+
+import pytest
+
+from wazuh_testing.tools import LOGCOLLECTOR_DAEMON, LOG_FILE_PATH
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.fim import callback_configuration_error
+
+
+# Marks
+pytestmark = [pytest.mark.tier(level=0), pytest.mark.agent]
+
+# Variables
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+files = ['test.txt']
+
+# Configuration
+daemons_handler_configuration = {
+    'daemons': [LOGCOLLECTOR_DAEMON],
+    'ignore_errors': True
```

Comment:
Using this structure is not possible to check if adding invalid configuration in `agent.conf` the agent still running.